app/code/community/Bolt/Boltpay/Controller/Traits/ApiControllerTrait.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * Bolt magento plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2018 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+/**
+ * Trait Bolt_Boltpay_Controller_Traits_ApiControllerTrait
+ *
+ * Defines generalized actions associated with API calls to and from the Bolt server
```

Comment:
this is only for API call *from* Bolt server, no?

Reply:
I can improve the language here.  I should use the words "Web hooks" rather than Api.  I will move this to say "Defines generalized actions associated with web hooks."

This includes from (receiving and processing), and to (response).